### PR TITLE
Update delete-a-user.md

### PR DESCRIPTION
**Pull request by Houndtech**
Current link is in error - points to https://docs.microsoft.com/en-us/microsoft-365/security/office-365-security/exchange-online-protection-overview
The proposed link points to "Managing inactive Mailboxes" to match the link description

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: Office365-Admin/add-users/delete-a-user.md [hunk 1]
```diff
@@ -173,7 +173,7 @@ See the [Remove-MsolUser](https://go.microsoft.com/fwlink/?linkid=842230) PowerS
     
 - If you want to keep the user's email, **BEFORE** you delete the account, move the email to a different location. If you've already deleted the account: if it's been less than 30 days you can restore it, then move the email data, then delete the account. See [Get access to and back up a former user's data](get-access-to-and-back-up-a-former-user-s-data.md).
     
-- If you have an Enterprise subscription, like Office 365 Enterprise E3, you can preserve the mailbox data of a deleted Office 365 user account by turn it into an *inactive mailbox*. To learn more, see [Manage inactive mailboxes in Exchange Online](https://go.microsoft.com/fwlink/p/?linkid=848214).
+- If you have an Enterprise subscription, like Office 365 Enterprise E3, you can preserve the mailbox data of a deleted Office 365 user account by turn it into an *inactive mailbox*. To learn more, see [Manage inactive mailboxes in Exchange Online](https://docs.microsoft.com/en-us/microsoft-365/compliance/inactive-mailboxes-in-office-365).
```
Comment (JohanFreelancer9) [inline Office365-Admin/add-users/delete-a-user.md]:
```suggestion
- If you have an Enterprise subscription like Office 365 Enterprise E3, you can preserve the mailbox data of a deleted Office 365 user account by turning it into an *inactive mailbox*. To learn more, see [Manage inactive mailboxes in Exchange Online](https://docs.microsoft.com/microsoft-365/compliance/inactive-mailboxes-in-office-365).
```

**Comment by mypil** [comment]
@cmcatee-MSFT - PR has been copy edited and is ready for final review. 

Thank you.

cc: @kenwith

File: Office365-Admin/add-users/delete-a-user.md [hunk 2]
```diff
@@ -173,7 +173,7 @@ See the [Remove-MsolUser](https://go.microsoft.com/fwlink/?linkid=842230) PowerS
     
 - If you want to keep the user's email, **BEFORE** you delete the account, move the email to a different location. If you've already deleted the account: if it's been less than 30 days you can restore it, then move the email data, then delete the account. See [Get access to and back up a former user's data](get-access-to-and-back-up-a-former-user-s-data.md).
     
-- If you have an Enterprise subscription, like Office 365 Enterprise E3, you can preserve the mailbox data of a deleted Office 365 user account by turn it into an *inactive mailbox*. To learn more, see [Manage inactive mailboxes in Exchange Online](https://docs.microsoft.com/en-us/microsoft-365/compliance/inactive-mailboxes-in-office-365).
+- If you have an Enterprise subscription like Office 365 Enterprise E3, you can preserve the mailbox data of a deleted Office 365 user account by turning it into an *inactive mailbox*. To learn more, see [Manage inactive mailboxes in Exchange Online](https://docs.microsoft.com/microsoft-365/compliance/inactive-mailboxes-in-office-365).
```
Comment (cmcatee-MSFT) [inline Office365-Admin/add-users/delete-a-user.md]: Please remove **en-us/** from the URL

**Pull request merged**